src/xrGame/Level_start.cpp

```diff
@@ -154,6 +155,8 @@ bool CLevel::net_start2()
         }
         Server->SLS_Default();
         map_data.m_name = Server->level_name(m_caServerOptions);
+        if (map_data.m_name.size())
+            g_DiscordRPC.Update(DiscordRPC::DiscordStatusType::UpdateLevel, StringTable().translate(map_data.m_name).c_str());
```

Comment:
Needs Windows-specific `#ifdef` blocks like in src/xrGame/GamePersistent.cpp:
```
CMakeFiles/xrGame.dir/Level_start.cpp.o:Level_start.cpp:function CLevel::net_start2(): error: undefined reference to 'g_DiscordRPC'
CMakeFiles/xrGame.dir/Level_start.cpp.o:Level_start.cpp:function CLevel::net_start2(): error: undefined reference to 'DiscordRPC::Update(DiscordRPC::DiscordStatusType, char const*)'
```

Reply:
No, no. No ifdefs, this can work on Linux, just needs to be fixed for it :)